ops/pebble.py

```diff
@@ -3046,6 +3105,9 @@ def get_checks(
         """
         query: Dict[str, Any] = {}
         if level is not None:
+            # is this bad now that value could be 'unknown'?
+            # should we explicitly avoid it here?
+            # if level is not None and level.value is not CheckLevel.UNKNOWN:
             query['level'] = level.value
```

Comment:
Another case where it's not clear how to handle `CheckLevel.UNKNOWN` on the outgoing side.